main.py

```
@@ -1,20 +1,310 @@
+
+async def del_admin(update, context):
+    if update.effective_user.id != OWNER_ID: return
+    try:
+        uid = int(context.args[0])
+        admins = load(ADMINS_DB)
+        if uid in admins and uid != OWNER_ID: admins.remove(uid); save(ADMINS_DB, admins)
+        await update.message.reply_text("✅ Admin removed")
+    except: pass
+
+async def handle_upload(update, context):
+    if not is_admin(update.effective_user.id): return
```

Ruff (0.14.10)

Multiple statements on one line (colon)

(E701)